tools/qmctl/qmctl

```diff
@@ -274,48 +316,73 @@ class QMCTL:
 
 
 def main():
-    parser = argparse.ArgumentParser(
-        description="QMCTL: Manage QM container settings via Podman.",
-        epilog="""
-Examples:
-  ./qmctl show                             Show raw container config
-  ./qmctl show resources                   Show live systemd-cgtop for qm.service
-  ./qmctl show unix-domain-sockets        Show UNIX sockets inside container
-  ./qmctl show shared-memory              Show shared memory (ipcs)
-  ./qmctl show namespaces                 Show namespaces from container
-  ./qmctl show available-devices          Show devices from container config
-  ./qmctl show all                        Show all options + cgtop snapshot
-  ./qmctl exec uname -a                   Run 'uname -a' in container
-""",
-        formatter_class=argparse.RawTextHelpFormatter
-    )
-
-    parser.add_argument("--verbose", action="store_true", help="Show accessed file paths")
-
-    subparsers = parser.add_subparsers(dest="command", help="Available commands")
-
-    show_parser = subparsers.add_parser("show", help="Show container info")
-    show_parser.add_argument(
-        "topic",
-        nargs="?",
-        default="container",
-        choices=[
-            "container",
-            "unix-domain-sockets",
-            "shared-memory",
-            "resources",
-            "available-devices",
-            "namespaces",
-            "all"
-        ],
-        help="What to show"
-    )
-    show_parser.add_argument("--json", action="store_true", help="Output as JSON")
-
-    exec_parser = subparsers.add_parser("exec", help="Run command inside QM container")
-    exec_parser.add_argument("cmd", nargs=argparse.REMAINDER, help="Command to execute")
-    exec_parser.add_argument("--json", action="store_true", help="Output as JSON")
-
+    qmctl = qmctl_add_new_command.qmctl_command_class()
```

Comment:
qmctl_add_new_command seems good for a different patch, still not sure if needed. Can you please just add new command method into our code. ?

Reply:
Done. Thank you